deployment/sandbox/docker/wait-for-flyte.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -e
+
+# Ensure cluster is up and running
+timeout 180 sh -c "until k3s kubectl explain deployment &> /dev/null; do sleep 1; done"
+
+# Wait for flyte deployment
+k3s kubectl wait --for=condition=available deployment/datacatalog deployment/flyteadmin deployment/flyteconsole deployment/flytepropeller -n flyte --timeout=180s
+
+timeout 180 sh -c "until [[ $(k3s kubectl get daemonset envoy -n flyte -o jsonpath='{.status.numberReady}') -eq 1 ]]; do sleep 1; done"
```

Comment:
There appears to be an issue where the entrypoint gets stuck here. Trying to figure it out.

Reply:
Is this fixed now with the quotes?

Reply:
Yes it is!